update dashboard deployment (#1900)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md

    Enable: false
  Log:     # Dashboard run log settings.
    KeepDays: 7    # The number of days for keeping log.

>     Mode: console   # The save mode of the log. console means the service logs will be logged in webserver.log. file means the service logs will be logged in access.log, error.log, severe.log, slow.log and stat.log respectively.

Reviewer: ```suggestion
    Mode: console   # The save mode of logs, including console and file. console means the service logs are logged in webserver.log. file means the service logs are logged in access.log, error.log, severe.log, slow.log, and stat.log respectively.
```

    messageStore: 90 # It sets the number of days to keep alert messages, the value of which is 90 by default. 
  ```

### Uninstallation

To uninstall Dashboard Enterprise Edition, run the following command.

```
sudo dpkg -r <package_name>
```

> ## Manage Dashboard Service

Reviewer: ```suggestion
## Manage services in Dashboard
```
